### ISSUE TITLE
Option to not follow symlink files in Explorer

### ISSUE THREAD

**Issue by bland328** [microsoft/vscode#97030]
**tl;dr:** I'd like the option to treat symlink files as simple files in VS Code Explorer (perhaps an `explorer.followSymlinks` option?)

**Goal:**
I'm aware of the `search.followSymlinks` option, and I'm hoping for something like an `explorer.followSymlinks` option, which would make VS Code Explorer treat symlink files as simple files, rather than resolving/following them.

**Reasoning:**
Symlinks are files, and while plenty of people use symlinks to files and directories to organize projects/work environments _(and would therefore likely want them to act like the target files and directories)_, some people like me have projects that include symlink files _as_ project files.

For many of my projects, symlink files are simply project assets, so I certainly _don't_ want to resolve/follow any of them. I _especially_ don't want to follow symlinks to directories--this really muddies VS Code Explorer navigation of such projects.

**Ideas:**
Perhaps, if the appropriate option were set, symlink files _could_ still be very intentionally followed, with a right-click "Follow symlink" option or something to that effect.

**Comment by isidorn** [comment]
This is a fair feature request, however we do not have it on our immediate plan. Thus assigning to backlog and we might add this in the future. Thanks

**Comment by bland328** [comment]
Thanks very much for the attention to this. I totally understand that it is far from emergent, but think it would be a useful addition, as sometimes a file is just a file :)

**Comment by Lennonka** [comment]
Can I somehow upvote this request? I want the feature too :)
